link_shortener/commands/create.py

```diff
@@ -2,22 +2,22 @@
 Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
 Licensed under the MIT (Expat) License (see LICENSE in Documentation).
 '''
-import os
 import hashlib
+import os
 
-from link_shortener.models import links, salts
+from sqlalchemy import and_
 
 from link_shortener.core.exceptions import DuplicateActiveLinkForbidden
+from link_shortener.models import links, salts
 
 
 async def create_link(request, data):
     async with request.app.engine.acquire() as conn:
         trans = await conn.begin()
-        query = await conn.execute(links.select().where(
-            links.columns['endpoint'] == data['endpoint']
-        ).where(
+        query = await conn.execute(links.select().where(and_(
+            links.columns['endpoint'] == data['endpoint'],
             links.columns['is_active'] == True
```

Comment:
Tady sem to taky vlastně trochu naprasil, a pořád kvůli tomu řve i linter. Tak to možná ještě v rámci tohodle PR taky změň prosím.

Mělo by místo `links.columns['is_active'] == True` fungovat jenom `links.c.is_active`, popřípadě `~links.c.is_active` pro False, a zkus to teda s těma columns[]. Když to půjde jen přes links.c.is_active tak to necháme na jiný PR.

Nebo koukni do dokumentace Alchemy, třeba najdeš něco lepšího. Hodilo by se něco jako is_() operátor, ale nic takovýho sem nenašel.

Reply:
Nebo pokud nechceš tu vlnovku tak myslim že existuje _not() operátor

Reply:
Fixnute
a68e024ee3e14584b8a4e9744cf6bbde38983012